src/world/config/examine-data.ts

```diff
@@ -0,0 +1,55 @@
+import { readFileSync } from 'fs';
+import { logger } from '@runejs/logger/dist/logger';
+import { JSON_SCHEMA, safeLoad } from 'js-yaml';
+
+interface Examine {
+    id: number;
+    examine: string;
+}
+
+export class ExamineCache {
+    private readonly items: Map<number, Examine>;
+    private readonly npcs: Map<number, Examine>;
+    private readonly objects: Map<number, Examine>;
+
+    public constructor() {
+        logger.info('Parsing examine data...');
+        this.items = parseData('data/config/examine-item-data.yaml');
+        this.npcs = new Map<number, Examine>();
```

Comment:
there are _some_ examines for "monsters" in osrsbox-db, however the ID numbers do not correspond with 435 cache id numbers. For this reason, I've excluded these from this initial PR. Not sure how we want to approach these.